.travis.yml

```diff
@@ -18,6 +18,12 @@ before_script:
     - export SAXON_CP=/tmp/xspec/saxon/saxon9he.jar
     - wget -O ${SAXON_CP} http://central.maven.org/maven2/net/sf/saxon/Saxon-HE/${SAXON_VERSION}/Saxon-HE-${SAXON_VERSION}.jar
     - chmod +x ${SAXON_CP}
+    # install XML Calabash
+    - export XMLCALABASH_VERSION=1.1.15-97
+    - wget -O /tmp/xspec/xmlcalabash-${XMLCALABASH_VERSION}.zip https://github.com/ndw/xmlcalabash1/releases/download/${XMLCALABASH_VERSION}/xmlcalabash-${XMLCALABASH_VERSION}.zip 
+    - sudo apt-get install -y unzip
```

Comment:
Good point, it's better not to use ```sudo``` - if it can be avoided - for security reasons. I'll try the following things:

1. See if ```unzip``` is already installed by default so there is no need to install it
2. If 1 doesn't work, try to install it without ```sudo```
3. If 2 doesn't work, use the [recommended approach](https://docs.travis-ci.com/user/migrating-from-legacy/#How-do-I-install-APT-sources-and-packages%3F) for container-based infrastructure